docs/creating-a-theme.txt

```diff
@@ -1,384 +1,767 @@
+   This is just the base theme.
 
-    $ nikola build && nikola serve
+Basic CSS
+---------
 
-.. figure:: http://ralsina.me/galleries/monospace-tut/monospace-1.png
-   :height: 400px
+The next step is to know exactly how Lanyon's pages work. To do this, we read its HTML.
+First let's look at the head element:
 
-   This is the default "bootstrap" theme.
+.. code:: html
 
-Of course, the page layout is completely different from what we want. To fix that, we need to
-get into templates.
+    <!DOCTYPE html>
+    <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">
 
-Templates: Page Layout
-----------------------
+    <head>
+    <link href="http://gmpg.org/xfn/11" rel="profile">
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
 
-The general page layout for the theme is done by the ``base.tmpl`` template, which is done using
-`Mako <http://www.makotemplates.org/>`_. This is bootstrap's ``base.tmpl``, it's not very big:
+    <!-- Enable responsiveness on mobile devices-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
 
-.. code-block:: mako
+    <title>
+        Lanyon &middot; A Jekyll theme
+    </title>
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="/public/css/poole.css">
+    <link rel="stylesheet" href="/public/css/syntax.css">
+    <link rel="stylesheet" href="/public/css/lanyon.css">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400">
+
+    <!-- Icons -->
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-144-precomposed.thumbnail.png">
+    <link rel="shortcut icon" href="/public/favicon.ico">
+
+    <!-- RSS -->
+    <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
+
+    <!-- Google Analytics -->
+    [...]
+    </head>
+
+The interesting part there is that it loads a few CSS files. If you check the source of your Nikola site, you will
+see something fairly similar:
+
+.. code:: html
 
-    ## -*- coding: utf-8 -*-
-    <%namespace name="base" file="base_helper.tmpl" import="*" />
-    <%namespace name="bootstrap" file="bootstrap_helper.tmpl" import="*" />
-    ${set_locale(lang)}
     <!DOCTYPE html>
-    <html% if comment_system == 'facebook': xmlns:fb="http://ogp.me/ns/fb#" %endif lang="${lang}">
+    <html prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# " vocab="http://ogp.me/ns" lang="en">
     <head>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        ${bootstrap.html_head()}
-        <%block name="extra_head">
-        </%block>
-        ${extra_head_data}
+    <meta charset="utf-8">
+    <meta name="description" content="This is a demo site for Nikola.">
+    <meta name="viewport" content="width=device-width">
+    <title>My Nikola Site | My Nikola Site</title>
+
+    <link href="assets/css/rst.css" rel="stylesheet" type="text/css">
```

Comment:
In any case, I have no idea why this file is in this diff. It's exactly the same in master :-(